content/blog/data-processing-compliance-statements-and-sla.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Data Processing, Compliance Statements and SLA
+author: Arnout Engelen
+date: 2024-07-10
+---
+
+The Apache Software Foundation is a non commercial, open source organization that relies on a large community of volunteers and companies to maintain its software. It provides this software free (and gratis) 'as is' - as per [its license](https://www.apache.org/licenses/LICENSE-2.0).
+
+As such we are not a 'vendor'. For readers not yet familiar, it might be interesting to read more about [how the ASF works](https://apache.org/foundation/how-it-works.html).
```

Comment:
Maybe we should mention that we are not a vendor but "Open Source Steward" - I think it's about a good time to start promoting it and mention in as many places as possible - we have about 2 years for the ecosystem to get used to it and it nicely separates us out from "vendors".

Reply:
(I see it's mentioned below - cool - but mentioning it here in contrast to vendor might be still a good idea).